addons/gdUnit4/src/core/GdUnitSettings.gd

```diff
@@ -280,20 +283,29 @@ static func list_settings(category :String) -> Array[GdUnitProperty]:
 			var default :Variant = ProjectSettings.property_get_revert(property_name)
 			var help :String = property["hint_string"]
 			var value_set := extract_value_set_from_help(help)
-			settings.append(GdUnitProperty.new(property_name, property["type"], value, default, help, value_set))
+			settings.append(GdUnitProperty.new(property_name, property["type"], value, default, cleanup_help(help), value_set))
 	return settings
 
 
 static func extract_value_set_from_help(value :String) -> PackedStringArray:
+	var split_value := value.split(_VALUE_SET_SEPARATOR)
+	if not split_value.size() > 1:
+		return PackedStringArray()
+
 	var regex := RegEx.new()
 	regex.compile("\\[(.+)\\]")
-	var matches := regex.search_all(value)
+	var matches := regex.search_all(split_value[1])
 	if matches.is_empty():
 		return PackedStringArray()
-	var values :String =  matches[0].get_string(1)
+	var values: String = matches[0].get_string(1)
+	var help: String
```

Comment:
```suggestion
```

Reply:
Oops, leftover code I should have removed, fixed